tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -82,9 +82,6 @@ addQuarters
 addSeconds
 addWeeks
 addYears
-addressToLine
```

Comment:
Why are these functions getting removed? Current fast-test failures are due to this: https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=79634&sha=4924168f0fe35463b66d486fa48bc1778c7c8e79&name_0=PR&name_1=Fast+test&name_2=Tests

Reply:
I saw these but when I run that tests locally, this is what I get as an output. I can revert them to pass the test.